src/App.tsx

```diff
@@ -5,7 +5,15 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie) => {
+    if (movies.find(movieInList => movieInList.imdbId === movie.imdbId)) {
```

Comment:
will be better to store found movie in separate variable instead of call find method inside the if